### ISSUE TITLE
Cleanup scripts getting put in the wrong place

### ISSUE THREAD

**Issue by QuinnBast** [rancher/rke2#4152]
**Environmental Info:**
RKE2 Version: v1.26.3_rke2r1

Node(s) CPU architecture, OS, and Version: 
Linux hostname 5.14.0-162.6.1.e19_1.x86_64 #1 CMP PREEMPT_DYNAMIC Tue Nov 15 07:49:10 EST 2022 x86_64 x86_64 GNU/Linux

Cluster Configuration:
3 servers

**Describe the bug:**
The [docs here](https://docs.rke2.io/install/quickstart) state:

> Two cleanup scripts, `rke2-killall.sh` and `rke2-uninstall.sh`, will be installed to the path at:
    - `/usr/local/bin` for regular file systems
    - `/opt/rke2/bin` for read-only and brtfs file systems
    - `INSTALL_RKE2_TAR_PREFIX/bin` if `INSTALL_RKE2_TAR_PREFIX` is set
    
However, after running my installation, I was unable to find the uninstallers. It turns out that this was not the case:

> `INSTALL_RKE2_TAR_PREFIX/bin` if `INSTALL_RKE2_TAR_PREFIX` is set

I did an `echo $INSTALL_RKE2_TAR_PREFIX` and it returned empty/unset. But even though it was unset, the uninstall scripts got installed to `INSTALL_RKE2_TAR_PREFIX/bin`, which ended up just being `/bin`. This means that they are accessible from anywhere in the machine globally (potentially [related to this request](https://github.com/rancher/rke2/issues/3928))

**Steps To Reproduce:**
- Install RKE2 using tarball install

**Expected behavior:**
Uninstall scripts are in `/usr/local/bin`

**Actual behavior:**
Uninstall scripts are in `/bin`

**Comment by brandond** [comment]
I suspect you've installed rke2 from rpm, not from the tarball, is that correct? The docs currently only describe behavior for tarball installs.

cc @dereknola

**Comment by QuinnBast** [comment]
No, I am using the tarball installs.

I've got an ansible script setup to automate air-gapped installations and every time the uninstall scripts are getting placed in `/bin`. The process looks something like this:

- Download [core](https://github.com/rancher/rke2/releases/download/v1.26.3%2Brke2r1/rke2-images-core.linux-amd64.tar.gz) and [network](https://github.com/rancher/rke2/releases/download/v1.26.3%2Brke2r1/rke2-images-cilium.linux-amd64.tar.gz) and [installer script](https://get.rke2.io) to my local machine
- Copy/SCP the downloaded files into `/var/lib/rancher/rke2/agent/images/` on each remote host
- Run installer on first host `/var/lib/rancher/rke2/agent/images/install.sh`
- systemctl enable and start the rke2-server service

**Comment by brandond** [comment]
There are discussions here on this repo, and there's also the Rancher Users slack.

**Comment by brandond** [comment]
I can't replicate what you're reporting. The bin and scripts are extracted to the documented location:
```console
systemd-node-1:/ # curl -sL get.rke2.io | sh -s -
[INFO]  finding release for channel stable
[INFO]  using v1.25.9+rke2r1 as release
[INFO]  downloading checksums at https://github.com/rancher/rke2/releases/download/v1.25.9+rke2r1/sha256sum-amd64.txt
[INFO]  downloading tarball at https://github.com/rancher/rke2/releases/download/v1.25.9+rke2r1/rke2.linux-amd64.tar.gz
[INFO]  verifying tarball
[INFO]  unpacking tarball file to /usr/local

systemd-node-1:/ # ls /usr/local/bin/rke2*
/usr/local/bin/rke2  /usr/local/bin/rke2-killall.sh  /usr/local/bin/rke2-uninstall.sh

systemd-node-1:/ # ls /bin/rke2*
ls: cannot access '/bin/rke2*': No such file or directory
```

Are you setting any other variables that might influence the install script behavior? Can you run the install script as `/var/lib/rancher/rke2/agent/images/install.sh -x` and report the complete output?

**Comment by QuinnBast** [comment]
Apologies, it appears I was running the `./install.sh` script without setting `INSTALL_RKE2_ARTIFACT_PATH` when I intended to set this. After setting this, it turns out I needed to download and copy over a few additional files that were not mentioned in the documentation, otherwise the `./install.sh` script didn't work:

- [Checksum](https://github.com/rancher/rke2/releases/download/v1.26.3%2Brke2r1/sha256sum-amd64.txt)
- [Rke2 Binary](https://github.com/rancher/rke2/releases/download/v1.26.3%2Brke2r1/rke2.linux-amd64.tar.gz)

My servers are not truly air-gapped and do have internet connection, so they were initially fetching from the internet when I was running `./install.sh` instead of using the tarballs.

Now that I've set `INSTALL_RKE2_ARTIFACT_PATH` the uninstall scripts got put in the right place.

However, I suppose that this means that when using the `rpm` install method the uninstall scripts are going to the wrong spot.
